tests/DCPS/Compiler/xcdr/xcdr.cpp

```diff
@@ -132,6 +179,18 @@ ::testing::AssertionResult assert_values(
   return ::testing::AssertionSuccess();
 }
 
+template<typename TypeA, typename TypeB>
+::testing::AssertionResult assert_values_union(
+  const char* a_expr, const char* b_expr,
+  const TypeA& a, const TypeB& b)
+{
+  expect_values_equal_union(a, b);
+  if (::testing::Test::HasFailure()) {
+    return ::testing::AssertionFailure() << a_expr << " != " << b_expr;
+  }
+  return ::testing::AssertionSuccess();
+}
```

Comment:
This isn't needed given template specializations are provided for `expect_values_equal`.